intercom_flutter_web/example/integration_test/intercom_flutter_web_test.dart

```diff
@@ -84,8 +84,12 @@ void main() {
             name: 'John Doe',
             userId: '1',
             phone: '+37256123456',
-            company: 'Some Company LLC',
-            companyId: '2',
+            company: {
+              'company_id': 'Test id',
+              'name': 'Some Company LLC',
+              'Country code': 'IT',
+              'Product': 'Host',
```

Comment:
Minor. It will be good to have "Hostless" code here. Please, remove `Country code` and `Product`. Use something abstract like `custom_attribute_1`, `custom_attribute_2`...

Reply:
done, thanks man!